### PR TITLE
Update proxy annotations docs

### PR DESCRIPTION
**Pull request by alpeb**
Added some missing entries.

### CODE REVIEW

File: cli/cmd/doc.go [hunk 1]
```diff
@@ -149,11 +153,15 @@ func generateAnnotationsDocs() []annotationDoc {
 		},
 		{
 			Name:        k8s.ProxyIgnoreInboundPortsAnnotation,
-			Description: "Ports that should skip the proxy and send directly to the application",
+			Description: "Ports that should skip the proxy and send directly to the application. Comma-separated list of values, where each value can be a port number or a range `a-b`.",
+		},
+		{
+			Name:        k8s.ProxyOpaquePortsAnnotation,
+			Description: "Ports that skip the proxy's protocol detection mechanism and are proxied opaquely",
```
Comment (wmorgan) [inline cli/cmd/doc.go:160]: Does it make sense to have the same format description here too? "Comma-separated list of...."

Comment (alpeb) [reply cli/cmd/doc.go]: Indeed :+1:

**Comment by alpeb** [comment]
Also fixes #5894

**Review by dadjeibaah — APPROVED** [review]
📦 🚢

**Review by wmorgan — APPROVED** [review]
I only reviewed the text but it LGTM. Thanks for updating all this.

**Comment by adleong** [comment]
Fixes #5894

**Review by adleong — APPROVED** [review]

**Pull request merged**